Makefile

```diff
@@ -6,6 +6,8 @@ test:
 .PHONY: e2e-test
 e2e-test:
 	# test all packages
+	go clean -testcache
```

Comment:
For local development, this is quite handy, since attempting to run the entire integration test suite, after changing only a JS test file, is impossible due to Go caching.